lib/pure/algorithm.nim

```diff
@@ -556,6 +558,76 @@ proc isSorted*[T](a: openArray[T], order = SortOrder.Ascending): bool =
     assert isSorted(e) == false
   isSorted(a, system.cmp[T], order)
 
+proc merge*[T](
```

Comment:
`sortedMerge` or `mergeSorted`

Reply:
These two names are misleading, because users may mess up it with `mergeSort` algotithm.

Reply:
ok; and consistent with D and C++